pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -446,13 +446,29 @@ func configureQemuConf(qemuFilename string) (err error) {
 		return err
 	}
 
-	if envVarValue, ok := os.LookupEnv("VIRTIOFSD_DEBUG_LOGS"); ok && (envVarValue == "1") {
+	if debugLogsStr, ok := os.LookupEnv("VIRTIOFSD_DEBUG_LOGS"); ok && (debugLogsStr == "1") {
 		_, err = qemuConf.WriteString("virtiofsd_debug = 1\n")
 		if err != nil {
 			return err
 		}
 	}
 
+	if pathsStr, ok := os.LookupEnv(services.ENV_VAR_SHARED_FILESYSTEM_PATHS); ok {
```

Comment:
Don't we have the information on the VMI status?

Reply:
You don't have the access mode of the PVCs in the VMI, only the claim name

Reply:
/hold
What do you mean https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/vm.go#L2521 ?

Reply:
Correction, you have the access modes, you don't have the paths from the container point of view.

Reply:
I beg to differ, you have whole VMI, it has the status which contains the access modes and volumes that has the paths. Am I missing anything?

Reply:
ah yes Lubo, you are right.

Reply:
> Am I missing anything?

Yes, you don't have the path from the container's point of view, where the volume is mounted.

Reply:
The access mode in itself is not enough

Reply:
Following offline conversations, it's possible
(through very fragile replacements of volumes, that we may or may not keep in the future)
to access the path/access mode.
However, I am completely against that and would prefer to declare the env upfront without dependency on the VMI.

I also think this way we can also be slightly more confident that `shared_filesystems` is respected throughout all paths since it's setup before the call to `/usr/sbin/virtqemud`.

I think asking for a hold cancel is therefore fair at this point.

@xpivarc @alicefr

Reply:
@akalenyu is the path from the HostDisk right? If so, yes, I agree. We should remove that awful mechanism in the future.

Reply:
/hold cancel
My concern was that we are adding additional mechanism to pass information around. As this is needed before starting the virtqemud I don't find this as additional anymore. I don't have enough time if this is covering hotplug/migration and will trust others to verify.